macs/README.md

```diff
@@ -0,0 +1,4 @@
+# Deploying to darwin
+
+Currently all mac-mini's are housed in [Grahams](https://github.com/grahamc) house.
+[@cole-h](https://github.com/cole-h) can deploy to these machines.
```

Comment:
the mac minis at graham's are not currently only, but he's willing to send them anywhere for hosting.

we currently rely fully on rented mac minis at hetzner online for hydra.

Reply:
Ok. I will ask around if I can find a spot somewhere.

Reply:
(The x86-only mac-minis are offline; the arm64 ones should be online)

Reply:
Added also a remark about x86